force-app/tests/Invocable Recipes/InvocableMethodRecipes_Tests.cls

```diff
@@ -7,7 +7,7 @@ private class InvocableMethodRecipes_Tests {
         request.inputs = new List<Account>{ new Account(name = 'test') };
         Test.stopTest();
 
-        System.assertEquals(
+        System.Assert.areEqual(
```

Comment:
Fix param order

Reply:
Ah I see now that some of my commits were not pushed up! Thanks for catching it.